### ISSUE TITLE
Make window minimize to tray

### ISSUE THREAD

**Issue by christasich** [Inndy/TerminalSummoner#2]
This is a great little tool! I think it'll push me over from Cmder to Windows Terminal. Is it possible to make the terminal window hide or minimize to tray so it's not always present after being dismissed?

**Comment by ghost** [comment]
Yes, very cool and necessary additional functionality. I vote for!
